docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[troubleshooting]]
+== Troubleshooting
+
+Is something not working as expected?
+Don't worry if you can't figure out what the problem is; we’re here to help!
+As a first step, ensure your app is compatible with the agent's <<supported-tech,supported technologies>>.
+
+If you're an existing Elastic customer with a support contract, please create a ticket in the
+https://support.elastic.co/customers/s/login/[Elastic Support portal].
+Other users can post in the https://discuss.elastic.co/c/apm[APM discuss forum].
+
+IMPORTANT: *Please attach your debug logs* so that we can analyze the problem.
+Upload the *complete* logs to a service like https://gist.github.com.
+Logs should include everything from when the application starts up until the first request executes.
+Instructions for enabling logging are below.
+
+[float]
+[[agent-logging]
+=== Logging
+
+Agent logs are critical to the debugging process.
+By default, this logging is disabled. To enable it, set a log output file with <<config-log-file,`ELASTIC_APM_LOG_FILE`>>.
+Alternatively, if you're using Docker or Kubernetes and are okay with mixing agent and application logs,
+you can set `ELASTIC_APM_LOG_FILE=stderr`.
+
+NOTE: The agent does not rotate log files. Log rotation must be handled externally.
+
+With logging enabled, you can use <<config-log-level,`ELASTIC_APM_LOG_LEVEL`>> to increase the granularity of the agent's logging.
+For example: `ELASTIC_APM_LOG_FILE=debug`.
+
+Make sure to execute a few requests to your application before posting your log files.
+Each request should add lines similar to these in the logs:
+
+[source,log]
+----
+{"@timestamp":"2020-07-23T03:14:16Z","client-ip":"127.0.0.1","duration":31693200,"log.level":"info","message":"GET /api/orders (200)","method":"GET","path":"/api/orders","status":200,"trace.id":"9790410c7c9ff50ea1606094bf75946d","transaction.id":"9790410c7c9ff50e"}
```

Comment:
That log message isn't coming from the agent. Here's an example log:

```
{"level":"debug","time":"2020-07-23T11:46:32+08:00","message":"sent request with 100 transactions, 0 spans, 0 errors, 0 metricsets"}
```

Reply:
🤦 Thanks as always, Andrew!